packages/elements/src/account/elements/logto-username.ts

```diff
@@ -0,0 +1,35 @@
+import { html, type TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
+
+import usernameIcon from '../icons/username.svg';
+
+import { LogtoProfileItemElement } from './LogtoProfileItemElement.js';
+
+const tagName = 'logto-username';
+
+@customElement(tagName)
+export class LogtoUsername extends LogtoProfileItemElement {
```

Comment:
Add some jsDoc comments here? Briefly describe the usage of this class and maybe key properties?

Reply:
Will add all comments with other elements later.